### PR TITLE
Add support for testing python test classes

### PR DESCRIPTION
**Pull request by andrew-cybsafe**
This PR adds a TestClass command that will execute a Python test class with pytest and addresses issue #295. Before updating documentation and tests, I wanted to get some feedback on whether you are ok with this approach and if there's other work needed to get this to a mergeable state. Thanks!

Make sure these boxes are checked before submitting your pull request:

- [x] Add fixtures and spec when implementing or updating a test runner
- [x] Update the README accordingly
- [x] Update the Vim documentation in `doc/test.txt`

### CODE REVIEW

**Comment by andrew-cybsafe** [comment]
@codeinabox I've added tests, updated the readme, and updated the vim docs. So this PR is now ready for review.

File: spec/pytest_spec.vim [hunk 1]
```diff
@@ -53,6 +53,16 @@ describe "PyTest_xunit"
...
+
+    view +5 test_class.py 
+    TestClass
+
+    Expect g:test#last_command == 'python -m pytest test_class.py::TestSubclass'
+
+    view +6 test_class.py 
+    TestClass
+
+    Expect g:test#last_command == 'python -m pytest test_class.py::TestSubclass'
```
Comment (codeinabox) [inline spec/pytest_spec.vim:65]: Could these TestClass examples please be removed out of the TestNearest block

**Review by codeinabox — CHANGES_REQUESTED** [review]
Thank you for your hard work on this, one small change and I am happy to merge

File: spec/pytest_spec.vim (hunk 1 above)
Comment (andrew-cybsafe) [reply spec/pytest_spec.vim]: Moved.

**Review by codeinabox — APPROVED** [review]
Thank you!

**Pull request merged**